src/main/resources/view/DisplayCard.fxml

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.Separator?>
+<?import javafx.scene.layout.ColumnConstraints?>
+<?import javafx.scene.layout.FlowPane?>
+<?import javafx.scene.layout.GridPane?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.Region?>
+<?import javafx.scene.layout.RowConstraints?>
+<?import javafx.scene.layout.VBox?>
+
+
+<HBox alignment="CENTER" xmlns="http://javafx.com/javafx/21" xmlns:fx="http://javafx.com/fxml/1">
+   <children>
+      <HBox id="displayPane" fx:id="displayPane" maxHeight="1.7976931348623157E308" maxWidth="1.7976931348623157E308" minWidth="300.0" HBox.hgrow="ALWAYS">
+        <GridPane HBox.hgrow="ALWAYS" VBox.vgrow="ALWAYS">
+          <columnConstraints>
```

Comment:
I notice inconsistent casing in this file, can we standardize and put PascalCase for class names? So stuff like ColumnConstrains, Children, Padding, MinWidth, RowConstraints etc.